src/main/java/org/apache/commons/lang3/exception/ContextedRuntimeException.java

```diff
@@ -214,13 +214,13 @@ public Object getFirstContextValue(final String label) {
         return this.exceptionContext.getContextEntries();
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public Set<String> getContextLabels() {
-        return exceptionContext.getContextLabels();
-    }
+//    /**
+//     * {@inheritDoc}
+//     */
+//    @Override
+//    public Set<String> getContextLabels() {
+//        return exceptionContext.getContextLabels();
+//    }
```

Comment:
It's hard to understand why this was commented out. This is also a public method. We cannot remove it without a major release - https://semver.org/